app/src/main/res/values/strings.xml

```diff
@@ -15,7 +15,7 @@
   -->
 
 <resources>
-    <string name="app_name">Todo</string>
+    <string name="app_name">Tasks</string>
```

Comment:
It's always been the Todo app.

Reply:
I'll change it back - the app used to reference todo, tasks and notes. We picked tasks, so I thought we might want to change the app name as well.